### ISSUE TITLE
Incorrect query result with skip()/take() when items are sorted in descending order (possibly related to indices)

### ISSUE THREAD

**Issue by SilentAntenna** [appy-one/acebase#120]
Version 1.21.6.

Suppose that the form of each item looks like this:
```
{
  "name": "indexed_property",
  "age": "data" 
}
```
And we created an index for the `name` property:

```
db.indexes.create('myTable', 'name');
```

Then we construct a query in the following manner:

```
db.query('news').sort('name', false).take(5).skip(20);
```

The query will not skip items properly.

This is probably related to line 707-712 of `acebase/src/api-local.js`. We sort the items in a descending way but toss the first value every time.

**Comment by appy-one** [comment]
I published v1.21.7 which should have fixed this. Let me know!

[![Spread the word](https://user-images.githubusercontent.com/26569719/169265089-3d593555-e1ad-4390-986b-877ac2c38a47.svg)](https://twitter.com/intent/tweet?button=&url=https://github.com/appy-one/acebase&text=I'm+using+@AcebaseRealtime+in+my+project+to+make+my+life+easier!&button=)  [![contribute](https://user-images.githubusercontent.com/26569719/169265318-30c4c6a5-7c89-46a0-a7a2-ef433a8192f4.svg)](https://github.com/appy-one/acebase#contributing)  [![Sponsor AceBase](https://user-images.githubusercontent.com/26569719/168233053-8e56b243-4140-40ab-9a30-4cb3cc149bfe.svg)](https://github.com/sponsors/appy-one)

**Comment by SilentAntenna** [comment]
Problem fixed. Thanks!
